src/dmd/dinterpret.d

```diff
@@ -852,6 +865,7 @@ public:
         {
             printf("%s IfStatement::interpret(%s)\n", s.loc.toChars(), s.condition.toChars());
         }
+        incUsageCtfe(istate, s.loc);
```

Comment:
I.e. line 869 guarantees that `istate` is not null.

Reply:
istate is non-null in most cases, yes.
not in all cases, I was surprised myself that I had to check for `istate` being non-null.

Reply:
`incUsageCtfe` checks it for null. Should not have redundant checks.

Reply:
so you are saying I should remove the check in `incUsage`  ?
If I do that the PR fails.
There are cases in which istate is null.